forum/users/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+from django.contrib.auth.password_validation import validate_password
+from .models import User
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(
+        write_only=True,
+        validators=[validate_password]
+    )
+
+    class Meta:
+        model = User
+        fields = ['email', 'password', 'user_name', 'first_name', 'last_name', 'role']
+
+
+    def validate_email(self, value):
+        if User.objects.filter(email=value).exists():
```

Comment:
In your validate_email, consider making the comparison case-insensitive:
 if User.objects.filter(email__iexact=value.lower()).exists():